test/box/fix-not-deleting-unix-socket-path.result

```diff
@@ -0,0 +1,94 @@
+-- test-run result file version 2
+env = require('test_run')
+ | ---
+ | ...
+net_box = require('net.box')
+ | ---
+ | ...
+fio = require('fio')
+ | ---
+ | ...
+test_run = env.new()
+ | ---
+ | ...
+test_run:cmd("create server test with script='box/fix-not-deleting-unix-socket-path.lua'")
+ | ---
+ | - true
+ | ...
+
+-- Check that unix socket path deleted after tarantool is finished
+test_run:cmd("setopt delimiter ';'")
+ | ---
+ | - true
+ | ...
+for i = 1, 2 do
+    local thread_count = i
+    test_run:cmd(string.format("start server test with args=\"%s\"", thread_count))
+    server_addr = test_run:cmd("eval test 'return box.cfg.listen'")[1]
+    test_run:cmd("stop server test")
+    assert(fio.path.exists(server_addr) == false)
+end
+test_run:cmd("setopt delimiter ''");
+ | ---
+ | ...
+
+-- Check, that all sockets are closed correctly,
+-- when the listening address is changed.
+test_run:cmd(string.format("start server test with args=\"%s\"", 2))
+ | ---
+ | - true
+ | ...
+server_addr_before = test_run:cmd("eval test 'return box.cfg.listen'")[1]
+ | ---
+ | ...
+test_run:eval("test", string.format("box.cfg{ listen = \'%s\' }", server_addr_before .. "X"))
+ | ---
+ | - []
+ | ...
+server_addr_after = test_run:cmd("eval test 'return box.cfg.listen'")[1]
+ | ---
+ | ...
+assert(server_addr_after == (server_addr_before .. "X"))
+ | ---
+ | - true
+ | ...
+assert(test_run:grep_log("test", "Bad file descriptor") == nil)
+ | ---
+ | - true
+ | ...
+assert(test_run:grep_log("test", "No such file or directory") == nil)
+ | ---
+ | - true
+ | ...
+test_run:eval("test", string.format("box.cfg { listen = \'%s\' }", server_addr_before))
+ | ---
+ | - []
+ | ...
+server_addr_result = test_run:cmd("eval test 'return box.cfg.listen'")[1]
+ | ---
+ | ...
+assert(server_addr_result == server_addr_before)
+ | ---
+ | - true
+ | ...
+test_run:cmd("stop server test")
+ | ---
+ | - true
+ | ...
+assert(fio.path.exists(server_addr_before) == false)
```

Comment:
For booleans you can use `not` operator.

Reply:
Fixed